db/migrate/20160831100250_add_field_type_to_custom_attributes.rb

```diff
@@ -0,0 +1,5 @@
+class AddFieldTypeToCustomAttributes < ActiveRecord::Migration[5.0]
+  def change
+    add_column :custom_attributes, :field_type, :string, :default => "String"
```

Comment:
It would be better to make the default "string", all lover case, because that's what the expression code expects. See [here](https://github.com/ManageIQ/manageiq/blob/1b50d729036443100e552f87dee59ca14779d4ba/lib/miq_expression.rb#L959-L959)